app/javascript/src/controllers/plan_controller.js

```diff
@@ -33,8 +33,8 @@ export default class extends Controller {
           currentTarget.value
         ]
       })
+      $.rails.fire($(currentTarget.closest("form")), "submit")
```

Comment:
this is necessary because `form.submit()` (the default browser way to submit a form) submits a plan HTML format request, instead of an XHR.

Reply:
What exactly does this do? It doesn't appear to save to the server, but there is a mysterious XHR request to localhost:3000 whenever I add an activity.